examples/io/plot_read_events.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _ex_read_events:
```

Comment:
it seems to add this in the rst file that's generated:

`.. _sphx_glr_auto_examples_io_plot_read_events.py`

that's quite ugly / a lot of text to add to our docstring links, no?